src/components/http/request/_mocks/Request.mock.xml

```diff
@@ -6,6 +6,8 @@
     <field id="data" type="assocarray" />
     <field id="options" type="assocarray" />
     <field id="result" type="node" />
+    <!-- Task fields -->
+    <field id="control" type="string" />
```

Comment:
Why this?

Reply:
This is a fix for tests - createRequest test sets `control` field which is not present in `RequestMock`, because it is an extension of a Node, not a Task.

Reply:
I mean, there were no failing tests but a lot of warnings that it tries to set non-existing field.